lte/gateway/c/core/oai/lib/itti/signals.c

```diff
@@ -158,35 +155,26 @@ int signal_handle(int* end, task_zmq_ctx_t* task_ctx) {
     perror("sigwait");
     return ret;
   }
-  // printf("Received signal %d\n", info.si_signo);
 
   /*
-   * Real-time signals are non constant and are therefore not suitable for
-   * * * use in switch.
+   * Dispatch the signal to sub-handlers
    */
-  if (info.si_signo == SIGTIMER) {
-    timer_handle_signal(&info, task_ctx);
-  } else {
-    /*
-     * Dispatch the signal to sub-handlers
-     */
-    switch (info.si_signo) {
-      case SIGABRT:
-        SIG_DEBUG("Received SIGABORT\n");
-        backtrace_handle_signal(&info);
-        break;
-
-      case SIGINT:
-      case SIGTERM:
-        printf("Received SIGINT or SIGTERM\n");
-        send_terminate_message_fatal(task_ctx);
-        *end = 1;
-        break;
-
-      default:
-        SIG_ERROR("Received unknown signal %d\n", info.si_signo);
-        break;
-    }
+  switch (info.si_signo) {
+    case SIGABRT:
+      SIG_DEBUG("Received SIGABORT\n");
+      backtrace_handle_signal(&info);
```

Comment:
Is the SIGABRT signal handler useful? It seems to only print the backtrace of the signal handler.

Reply:
The PR was focused on the timer, so I did not do any analysis on this. But in syslog I think this provides useful signals.